src/view/Dialog.less

```diff
@@ -0,0 +1,56 @@
+@import (reference) '~nav-frontend-core/less/core';
+
+.dialog {
+  background: @navLysGra;
+  display: flex;
+  flex-direction: column;
+}
+
+.dialog-top {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-content: space-around;
+  background: @navLysGra;
+  margin: 10px;
+
+}
+
+.dialog-bottom {
+  flex-grow: 1;
+  background: @navLysGra;
+ }
+
+.aktivitet {
+  flex-grow: 1;
```

Comment:
Denne klassen ser ikke ut til å ha noen effekt?

Reply:
tatt vekk tidligere, men trenges kanskje for margin under?